lib/ansible/cache/etcd.py

```diff
@@ -0,0 +1,82 @@
+# (c) 2014, Michael Scherer
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from __future__ import absolute_import
+import json
+
+from ansible import constants as C
+from ansible.cache.base import BaseCacheModule
+
+try:
+    from etcd.client import Client
+except ImportError:
+    import sys
+    print "The 'etcd' python module is required, 'pip install etcd'"
```

Comment:
actually, it should raise an AnsiblError exception (I have fixing the other cache plugins to do same on my list)